.releaserc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    branch: "master",
+    plugins: [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      ["@semantic-release/npm", { npmPublish: false }],
```

Comment:
false publish isn't needed (can be removed)

Reply:
[Removed](https://github.com/UniversalDataTool/udt-labelme-converter/pull/2/commits/793c5b5ca975ddad4c46a546ea43ea236ccb78ab)